### PR TITLE
Staging flavor

### PR DESCRIPTION
**Pull request by ValeriyMakarshin**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,31 +1,21 @@
...
 apply plugin: 'io.fabric'
 
-repositories {
-  maven { url 'https://maven.fabric.io/public' }
-}
 
 android {
   compileSdkVersion 25
-  buildToolsVersion "24.0.3"
+  buildToolsVersion '25.0.0'
```
Comment (RamilGabdrakhmanov) [inline app/build.gradle]: why you update buildToolsVersion?

File: app/src/staging/google-services.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (RamilGabdrakhmanov) [inline app/src/staging/google-services.json]: project use FCM?

File: app/src/staging/java/com/jrummy/busybox/installer/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,229 @@
+/*
```
Comment (RamilGabdrakhmanov) [inline app/src/staging/java/com/jrummy/busybox/installer/MainActivity.java]: this is copy from free?

File: app/src/staging/res/layout/activity_busybox_main.xml [hunk 4]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (RamilGabdrakhmanov) [inline app/src/staging/res/layout/activity_busybox_main.xml]: this is copy from free?

File: app/src/staging/res/menu/main_menu.xml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<!--
```
Comment (RamilGabdrakhmanov) [inline app/src/staging/res/menu/main_menu.xml]: this is copy from free?

File: build.gradle [hunk 6]
```diff
@@ -1,10 +1,12 @@
 buildscript {
   repositories {
     jcenter()
+    maven { url 'https://maven.fabric.io/public' }
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:2.2.3'
+    classpath 'com.android.tools.build:gradle:2.3.3'
```
Comment (RamilGabdrakhmanov) [inline build.gradle]: it is necessary?

File: app/build.gradle [hunk 7]
```diff
@@ -65,7 +67,14 @@ android {
   }
 }
 
+repositories {
+  maven { url 'https://maven.fabric.io/public' }
```
Comment (RamilGabdrakhmanov) [inline app/build.gradle]: it is necessary?

File: gradle/wrapper/gradle-wrapper.properties [hunk 8]
```diff
@@ -1,6 +1,6 @@
-#Wed Aug 24 16:54:37 PDT 2016
+#Mon Jul 10 17:41:50 MSK 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```
Comment (RamilGabdrakhmanov) [inline gradle/wrapper/gradle-wrapper.properties]: necessary?

File: app/src/staging/res/menu/main_menu.xml (hunk 5 above)
Comment (ValeriyMakarshin) [inline app/src/staging/res/menu/main_menu.xml]: yes

File: app/src/staging/res/layout/activity_busybox_main.xml (hunk 4 above)
Comment (ValeriyMakarshin) [inline app/src/staging/res/layout/activity_busybox_main.xml]: yes

File: app/src/staging/java/com/jrummy/busybox/installer/MainActivity.java (hunk 3 above)
Comment (ValeriyMakarshin) [inline app/src/staging/java/com/jrummy/busybox/installer/MainActivity.java]: yes

File: app/src/staging/google-services.json (hunk 2 above)
Comment (ValeriyMakarshin) [inline app/src/staging/google-services.json]: +

File: app/build.gradle (hunk 1 above)
Comment (RamilGabdrakhmanov) [inline app/build.gradle]: ?

Comment (RamilGabdrakhmanov) [inline app/build.gradle]: If it is not necessary, downgrade back

File: app/build.gradle (hunk 7 above)
Comment (RamilGabdrakhmanov) [inline app/build.gradle]: ?

File: app/build.gradle (hunk 1 above)
Comment (ValeriyMakarshin) [inline app/build.gradle]: Returned

File: app/build.gradle (hunk 7 above)
Comment (ValeriyMakarshin) [inline app/build.gradle]: Returned

File: app/src/staging/google-services.json (hunk 2 above)
Comment (ValeriyMakarshin) [inline app/src/staging/google-services.json]: yep

**Pull request merged**